build-index/tagmodels/models.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2016-2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package tagmodels
+
+const (
+	// Filters.
+	LimitQ  string = "limit"
+	OffsetQ string = "offset"
+)
+
+// List Response with pagination. Models tagserver reponse to list and
+// listRepository.
+type ListResponse struct {
```

Comment:
i'm still leaning towards unifying the two response structs. Also, looks like we parse the responses twice here to get offset and next. Is it possible to have `ListResponse.GetOffset()` and `ListResponse.GetNext()`?

Reply:
We are parsing twice for the reason above.